sys/include/xtimer.h

```diff
@@ -448,7 +448,14 @@ int xtimer_mutex_lock_timeout(mutex_t *mutex, uint64_t us);
  * @param[in]   t       timer struct to use
```

Comment:
you forgot to update the param name in the doxygen comment

Reply:
fixed